AppController/djinn.rb

```diff
@@ -4093,6 +4127,38 @@ def rsync_files(dest_node)
     HelperFunctions.shell("rsync #{options} #{lib}/* root@#{ip}:#{lib}")
     HelperFunctions.shell("rsync #{options} #{app_task_queue}/* root@#{ip}:#{app_task_queue}")
     HelperFunctions.shell("rsync #{options} #{scripts}/* root@#{ip}:#{scripts}")
+
+    # Run a build on modified directories so that changes will take effect.
+    get_status = 'git -C appscale status'
+    ssh_opts = "-i #{ssh_key} -o StrictHostkeyChecking=no " +
+      '-o NumberOfPasswordPrompts=0'
+    status = `ssh #{ssh_opts} root@#{ip} #{get_status}`
+
+    if status.include?('AppTaskQueue')
+      Djinn.log_info("Building uncommitted taskqueue changes on #{ip}")
+      build_tq = 'pip install --upgrade --no-deps ' +
+        "#{APPSCALE_HOME}/AppTaskQueue[celery_gui]"
```

Comment:
Can additional requirements like celery_gui be a constant somewhere? Dependencies might change over time.